### PR TITLE
Subscriptions Widget: Better subscribe failure UX

### PR DESCRIPTION
**Pull request by dagobertrenouf**
Improved user experience of subscribe widget when the form submission fails. Full details here: https://wordpress.org/support/topic/fix-proposal-subscription-widget-form-disappears-after-error?replies=2

### CODE REVIEW

**Comment by richardmtl** [comment]
Nice, thanks for the PR! A developper will take a look at it and merge it in if it's all good. :)

File: modules/subscriptions.php [hunk 1]
```diff
@@ -635,7 +635,7 @@ function widget( $args, $instance ) {
 		} else { ?>
 			<form action="#" method="post" accept-charset="utf-8" id="subscribe-blog-<?php echo $widget_id; ?>">
 				<?php
-				if ( ! isset ( $_GET['subscribe'] ) ) {
+				if ( ! isset ( $_GET['subscribe'] ) OR $_GET['subscribe'] != 'success' ) {
```
Comment (dereksmart) [inline modules/subscriptions.php]: Just a couple nitpicky code style requests: 

WP coding styles calls for <a href="https://make.wordpress.org/core/handbook/coding-standards/php/#yoda-conditions">yoda style conditions</a>.   We also typically use pipes for our logical operators.

**Comment by dereksmart** [comment]
Good idea.  Will be happy to merge once minor coding styles are updated

**Comment by dagobertrenouf** [comment]
@dereksmart just fixed it, let me know if it's good now.

**Comment by dereksmart** [comment]
@dagobertrenouf looks good, thanks! Merging.

**Pull request merged**